packages/compass-preferences-model/src/preferences.ts

```diff
@@ -392,6 +393,21 @@ const modelPreferencesProps: Required<{
       long: 'Hide credentials in connection strings from users.',
     },
   },
+  /**
+   * Switch to show the Kerberos password field in the connection form.
+   */
+  showKerberosPasswordField: {
+    type: 'boolean',
+    required: false,
+    default: false,
+    ui: true,
+    cli: true,
+    global: true,
+    description: {
+      short: 'Show Kerberos Password Field',
+      long: 'Shows the field in those rare situations in which Kerberos requires a password.',
```

Comment:
I think it might be a bit confusing to refer to “those rare situations” without actually saying what those situations are, and “in which Kerberos requires a password” might make it sound as if the protocol itself requires a password sometimes (which it does not). How about this?

```suggestion
      long: 'Show a password field for Kerberos authentication. Typically only useful when attempting to authenticate as another user than the current system user.',
```

if that isn’t too verbose?

Reply:
Sounds good, thx!